apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -898,12 +917,12 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
                     data["params"][answer_name], data["correct_answers"][answer_name]
                 )
             )
-            feedback = construct_feedback(
+            feedback += construct_feedback(
```

Comment:
Help me understand why this was changed?

Reply:
I forgot to delete "+" during the edition. I contacted Prof.Poulsen last week, and I know he won't be able to take a look until the end of August. I have read your messages above and may make a new pull request with a few changes.

Reply:
There's no need to open a brand new PR! You can push any changes to the same branch as part of this same PR.

Reply:
Thank you for the information. I am an undergraduate student at the University of Michigan. I have shown my professor this feature, and he provided some valuable feedback. Based on his advice, I will make additional enhancements to the feature. I understand it may take some time for Proof Blocks to officially incorporate my contributions, but I am passionate about participating in this development process.